COMMUNITY.md

```diff
@@ -296,6 +296,17 @@ submitting a new file, it is recommended to run that script over the new file
 after ensuring that the latest stable release of `clang-format` is in your
 PATH.
 
+LLVM also provides integrations for emacs, vim and git. The emacs and vim
+integrations are recommended as they make proper styling effortless. The git
+integration is strongly recommended; it can format commit ranges or pending
+changes and will only touch changed lines, which is very convenient. These
+tools are documented on the LLVM website and can be found in the LLVM trunk
+under `tools/clang/tools/clang-format`.
+
+When performing formatting tasks with `clang-format` please ensure you are
+using the latest offical version (i.e. the latest version available on
+[http://releases.llvm.org/download.html](http://releases.llvm.org/download.html))
+
```

Comment:
So does this mean we're going to reformat the code every time there is a version change.  I suspect not.  We should specify a specific version or later.

Reply:
Why?

Reply:
Please add a comment or parenthetical on the latest version that has been tested against and is known good.  Breakage does happen on new releases...